Cython/Debugger/Tests/test_libcython_in_gdb.py

```diff
@@ -175,6 +175,12 @@ def test_break_lineno(self):
         self.lineno_equals(nextline)
         assert step_result.rstrip().endswith(nextline)
 
+    def test_break_completion(self):
+        completer = libcython.cy.break_.complete
+        assert 'spam' in completer("codefile.SomeClass.s", "s")
+        assert 'spam' in completer("codefile.SomeClass.", None)
+        assert 'pam' in completer("codefile.s", None)
```

Comment:
It's difficult to debug failures if the output is just "assertion failed".
```suggestion
        self.assertIn('spam', completer("codefile.SomeClass.s", "s"))
        self.assertIn('spam', completer("codefile.SomeClass.", None))
        self.assertIn('pam', completer("codefile.s", None))
```

Reply:
I'm fine with that, I just used assert since that was what was used in the rest of the file. I was hoping for a more end to end test of tab completion but settled on this since it would require digging up a good amount of GDB source. Should I add more detailed explanations? Nothing I can find in the GDB API docs describe what the behavior should be.

Reply:
Actually, I'm going to revert this pending consistency with the rest of the tests

Reply:
It's a mix currently that is worth cleaning up at some point. But we shouldn't add to the mess from the wrong end.